### PR TITLE
docs: DLT-1933 move storybook to doc site - Item layout to Lazy show

### PR DESCRIPTION
**Pull request by ninamarina**
# Move storybook to doc site - Item layout to Lazy show

Components:

- Item layout: updated
- Keyboard shortcut: updated
- Lazy show: updated

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket
[DLT-1933]

[DLT-1933]: https://dialpad.atlassian.net/browse/DLT-1933?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/dialtone-documentation/docs/components/lazy-show.md
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/lazy-show.md:1]: Seems like the transition is not working? bug?

File: apps/dialtone-documentation/docs/components/keyboard-shortcut.md [hunk 1]
```diff
@@ -56,6 +65,89 @@ vueCode='
...
+:htmlCode="() => $refs.inlineExample"
+vueCode='
+Press
+<dt-keyboard-shortcut
+  screen-reader-text="Ctrl and F5"
+  shortcut="Ctrl + F5"
+/>
+to hard refresh the page.
+'
+/>
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/components/keyboard-shortcut.md:126]: Hmm this one seems to be rendering the screen-reader-text.

Screenshot 2024-08-07 at 1 47 26 PM

File: apps/dialtone-documentation/docs/components/lazy-show.md
Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/lazy-show.md:1]: Yeah, so weird it's not working in the doc site, it does in storybook. Looking into it

Comment (ninamarina) [reply apps/dialtone-documentation/docs/components/lazy-show.md:1]: We will fix this here: https://dialpad.atlassian.net/browse/DLT-1952

**Review by braddialpad — APPROVED** [review]
Great thanks!!

**Pull request merged**